src/Analyzers/CSharp/Tests/PopulateSwitch/PopulateSwitchStatementTests.cs

```diff
@@ -1224,7 +1224,7 @@ enum MyEnum
 }");
         }
 
-        [Fact, WorkItem(61281, "https://github.com/dotnet/roslyn/issues/61281")]
+        [Fact, WorkItem(61281, "https://github.com/dotnet/roslyn/pull/61809")]
```

Comment:
Although I hate this attribute since it has the value duplicated, update the first argument too?

Reply:
We also don't usually point this attribute to PRs at all, git blame will happily tell us when it was added. You could just delete the WorkItem attribute entirely if it's just wrong.

Reply:
I didn't create a separate issue and described in the PR itself what it is supposed to fix, so I think we should leave this `WorkItem`s. I'll update the first argument.